src/components/shared/__tests__/AddRemoveForm.spec.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+import AddRemoveForm from '../AddRemoveForm'
+import { shallow } from 'enzyme'
+import mockData from 'mockData'
+
+describe('<AddRemoveForm />', () => {
+  let wrapper
+  const mockProps = {
+    terms: {
+      title: 'Mock Terms',
+      // labelProp: 'name', // default
+      fetchItems: () => mockData.terms.getAll()
+    },
+    questions: {
+      title: 'Mock Questions',
+      labelProp: 'title',
+      fetchItems: () => mockData.questions.getAll()
+    }
+  }
```

Comment:
Since `mockProps` are specific to this component, we don't need it on our `mockData`. We may refactor it to the `mockData` later on if we can reuse the info across components.